index.html

```diff
@@ -31,6 +32,130 @@ <h1>Marco Almada</h1>
 
     <img src="images/image_geometry_2.svg" alt="" class="image-down" />
 
+    <section class="projects">
+        <h2>Projects</h2>
+    </section>
+    <section class="box">
```

Comment:
You have used too many `section` tags by creating one for each card.

- [ ] Make sure all the cards nest inside the `projects` section
- [ ] Use `display: grid` to align them
- [ ] Use `div` instead of `section` for `box`
- [ ] All the boxes/cards have similar styling, so you can reduce the code by creating one class and using that call for all the boxes/cards
- [ ] Use `display: flex` to align content of the box/card